fish_completions.go

```diff
@@ -53,6 +53,19 @@ function __%[1]s_perform_completion
     __%[1]s_debug "Calling $requestComp"
 
     set results (eval $requestComp 2> /dev/null)
+
+    # Some programs may output extra empty lines after the directive.
+    # Let's ignore them or else it will break completion.
+    # Ref: https://github.com/spf13/cobra/issues/1279
+    for line in $results[-1..1]
+        if test (string trim $line) = ""
```

Comment:
```suggestion
        if test (string trim -- $line) = ""
```

in case a line starts with a dash (unlikely but you never know)